modules/virt-aws-bm.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * virt/install/preparing-cluster-for-virt.adoc
+
+:_content-type: CONCEPT
+[id="virt-aws-bm_{context}"]
+= {VirtProductName} on AWS bare metal
+
+You can run {VirtProductName} on an Amazon Web Services (AWS) bare metal {product-title} cluster. Before you set up your cluster, review the following summary of supported features and limitations:
+
+Installing::
+
+* You can install the cluster by using installer-provisioned infrastructure, ensuring that you specify bare metal node instance types in the `install-config.yaml` file. For example, you can use the `c6g.metal` type value for a machine based on 64-bit ARM architecture.
+** For more information, see the {product-title} documentation about installing on AWS.
+
+Accessing virtual machines (VMs)::
+
+* There is no change to how you access VMs by using the `virtctl` CLI tool or the {product-title} web console.
+* You can expose VMs by using a `NodePort` or `LoadBalancer` service. 
+** The load balancer approach is preferable because {product-title} automatically creates the load balancer in AWS and manages its lifecycle. A security group is also created for the load balancer, and you can use annotations to attach existing security groups. When you remove the service, {product-title} removes the load balancer and its associated resources.
+
+Networking::
+
+* Because connecting a secondary network to a VM is not supported on AWS, you must use the primary network, which you can expose by using services. 
+* If your application requires a flat layer 2 network or control over the IP pool, consider using OVN-Kubernetes secondary overlay networks.
+* You cannot use Single Root I/O Virtualization (SR-IOV) or bridge Container Network Interface (CNI) networks, including virtual LAN (VLAN).
```

Comment:
Can we add a weird about *why* this is currently impossible? @AlonaKaplan

Reply:
@ronensdeor @AlonaKaplan  how about having a single bulletpoint saying:

* You cannot use Single Root I/O Virtualization (SR-IOV) or bridge Container Network Interface (CNI) networks, including virtual LAN (VLAN).  If your application requires a flat layer 2 network, consider using OVN-Kubernetes secondary overlay networks.

Reply:
I took this suggestion in my latest commit.